kuma/javascript/src/header/header.jsx

```diff
@@ -163,41 +167,42 @@ export default function Header(): React.Node {
     }
 
     return (
-        <div css={styles.header}>
-            <a css={styles.logoContainer} href={`/${locale}/`}>
-                <Logo css={styles.logo} alt="MDN Web Docs Logo" />
-            </a>
-            <Row css={styles.menus}>
-                {menus.map((m, index) => (
-                    <Dropdown label={gettext(m.label)} key={index}>
-                        {m.items.map((item, index) => (
-                            <li key={index}>
-                                {item.external ? (
-                                    <a
-                                        target="_blank"
-                                        rel="noopener noreferrer"
-                                        href={fixurl(item.url)}
-                                    >
-                                        {gettext(item.label)} &#x1f310;
-                                    </a>
-                                ) : (
-                                    <a href={fixurl(item.url)}>
-                                        {gettext(item.label)}
-                                    </a>
-                                )}
-                            </li>
-                        ))}
-                    </Dropdown>
-                ))}
-                <Spring />
-                <LanguageMenu />
-            </Row>
-            <div css={styles.search}>
-                <Search />
-            </div>
-            <div css={styles.login}>
-                <Login />
+        <>
+            <div css={styles.loadingBar} />
```

Comment:
The only change here is that the solid color blue bar used to just be a border. Now it is a div of its own, so we're adding the div and wrapping in `<>` and `</>`, which changes the indentation of everything.